falcon/routing/converters.py

```diff
@@ -83,15 +87,54 @@ def convert(self, value):
         except ValueError:
             return None
 
+        return self.validateMinMaxValue(value)
+
+    def validateMinMaxValue(self, value):
         if self._min is not None and value < self._min:
             return None
-
         if self._max is not None and value > self._max:
             return None
 
         return value
 
 
+class FloatConverter(IntConverter):
+    """Converts a field value to an float.
+
+    Identifier: `float`
+
+    Keyword Args:
+        num_digits (float): Require the value to have the given
```

Comment:
Indeed, as @CaselIT noted, `num_digits` doesn't really make sense for `float`s.
What is the number of digits in `2.9999999999999(9)`? `inf`? `1.23456789e123`?

We could either remove this parameter, or maybe introduce a different one? Like `significant_digits`? And then just round to the specified precision instead of failing validation.